src/commands/workflow/run.ts

```diff
@@ -44,4 +50,40 @@ export default class WorkflowRun extends BaseCommand {
       `‣ Successfully ran \`${args.workflowKey}\` workflow in \`${flags.environment}\` environment. Run id: ${resp.data.workflow_run_id}.`,
     );
   }
+
+  async runLocal(): Promise<void> {
+    const target = await Workflow.ensureValidCommandTarget(
+      this.props,
+      this.runContext,
+    );
+
+    const { type: targetType, context: targetCtx } = target;
+
+    if (!targetCtx.exists) {
+      const subject =
+        targetType === "workflowDir"
+          ? "a workflow directory at"
+          : "workflow directories in";
+
+      return CliUx.ux.error(
+        `Cannot locate ${subject} \`${targetCtx.abspath}\``,
+      );
+    }
+
+    if (targetType === "workflowDir") {
+      const [workflow] = await Workflow.readWorkflowDir(targetCtx, {
```

Comment:
It's possible for `readWorkflowDir()` to end up with errors (the second value in the returned tuple), so we probably should check that. 

Something like:
```
const [workflow, readErrors] = await Workflow.readWorkflowDir(targetCtx, { withExtractedFiles: true })

if (readErrors.length > 0) {
  this.error(formatErrors(readErrors));
}
```